src/test/java/org/broadinstitute/hellbender/tools/exome/AllelicCountUnitTest.java

```diff
@@ -0,0 +1,48 @@
+package org.broadinstitute.hellbender.tools.exome;
+
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+/**
+ * Created by davidben on 1/25/16.
+ */
+public class AllelicCountUnitTest {
+
+    @Test
+    public void testToMinorAlleleFraction() {
+        final SimpleInterval interval = new SimpleInterval("contig", 1, 2);
+        final double tolerance = 0.01;
```

Comment:
i think this tolerance should also be adjusted (see above) to make this test slightly more discerning, or a couple more test cases added.  for example, the naive min(a,r)/(a+r) passes 2/3 of the test cases.  more accurate truth values from mathematica, etc. could be used.